frontend/packages/kubevirt-plugin/src/components/vm-disks/disk-row.tsx

```diff
@@ -81,11 +82,11 @@ const getActions = (
 
   const isTemplate = vmLikeEntity && !isVM(vmLikeEntity);
   if (disk.isEditingSupported(isTemplate)) {
-    actions.push(menuActionEdit);
+    actions.push(menuActionEdit(disk, vmLikeEntity, opts));
   }
 
-  actions.push(menuActionDelete);
-  return actions.map((a) => a(disk, vmLikeEntity, opts));
+  actions.push(menuActionDelete(disk, vmLikeEntity, opts));
+  return actions;
```

Comment:
IMO the map approach is better 
- less code duplication
- one will not forget to add it when adding another action